opendatahub-io/modelmesh-serving
# Manifest updates resulting from #174

#### Motivation
Update modelmesh-serving manifests in the `opendatahub/odh-manifests` directory to sync with the manifest changes in `config/` resulting from the commits included in [this sync](https://github.com/opendatahub-io/modelmesh-serving/pull/174)

#### Modifications
- Copied over the new 0.11.0 manifests over to `opendatahub/odh-manifests/model-mesh/odh-modelmesh-controller` 

#### Result
CRD version on the `main` branch is updated to 0.11.0

#### PR checklist

Checklist items below are applicable for development targeted to both fast and stable branches/tags
- [ ] FVT tests pass locally
- [x] If the PR adds a new container image or updates the tag of an existing image (not build within cpaas), is the corresponding change made in live-builder and cpaas-midstream to add/update the image tag in the operator CSV? Link the PRs if applicable

Checklist items below are applicable for development targeted to both fast and stable branches/tags
- [x] Tested modelmesh serving deployment with odh-manifests and ran odh-manifests-e2e tests locally 

#### Testing Instructions 
- Check out the PR from github and use the opendatahub quickstarts to deploy modelmesh serving using the manifests in `opendatahub/odh-manifests/model-mesh`. Ensure Model Serving functionality works as intended
